src/database/index.ts

```diff
@@ -51,19 +51,32 @@ const connectionOptions = (() => {
 let pool: Pool;
 let serverReady = false;
 
-setTimeout(() => {
-  pool = createPool({
-    connectTimeout: 60000,
-    trace: false,
-    ...connectionOptions,
-    typeCast,
-  });
+const mysqlInit = (() => {
+  setTimeout(() => {
+    pool = createPool({
+      connectTimeout: 60000,
+      trace: false,
+      ...connectionOptions,
+      typeCast,
+    });
 
-  pool.query(mysql_transaction_isolation_level, (err) => {
-    if (err) return console.error(`^3Unable to establish a connection to the database!\n^3[${err}]^0`);
-    console.log(`^2Database server connection established!^0`);
-    serverReady = true;
+    pool.query(mysql_transaction_isolation_level, (err) => {
+      if (err) {
+        console.error(`^3Unable to establish a connection to the database!\n^3[${err}]^0`);
+        delete pool;
```

Comment:
Prevents building for me because of Strict mode. Should be removed since `pool` is reassigned in the next function call anyways.
```suggestion
```